### ISSUE TITLE
some doubts

### ISSUE THREAD

**Issue by SmartPorridge** [NVlabs/PWC-Net#18]
Hi Deqing,

Thanks for sharing your code, it's really cool. I trained it on Chairs but I still have some doubts.

1. The model size of Caffe/model/pwc_net.caffemodel  and pwc_net_chairs_things.caffemodel  is not equal. One is 43.6M and one is 69.5M , meanwhile, the caffemodel I trained using your solver and train.prototxt is about 39M. But they can be used by the same pwc_net_test.prototxt...... I am confused. The smaller one is from pwc-net-small in your paper?

2. The multi-gpus training of Flownet2.0 code has something wrong with the training time. In order to training with multi gpus, I add all the layers to my own caffe-tools, and I test the AEPE of your two pretrained models on Chairs_test(640 image pairs), the result is :
        pwc_net.caffemodel                            AEE： 2.8355
        pwc_net_chairs_things.caffemodel      AEE：2.3099
Is this result correct?  If it's wrong, would you share your AEE eval tools? 

3. For 1 GPU experiments ,I trained 3 times using the flownet2.0 code, and my batchsize is 8, all settings is copied from your code. But the loss didn't converge for 2 times and only 1 times the loss converged to 20-40. When set the batchsize to 32, it converged to 25-40.  Is that normal?

Thank you for your patience!
Best Regards!

**Comment by deqings** [comment]
Hi SmartPorridge,

Thanks for your interest in our work.
1. pwc_net.caffemodel has been trained through Chairs->Things->Sintel+HD1K+KITTI2015. My observation is that the caffemodel file size increases with more diverse training data, i.e., the weights are less sparse. 
2. the results sounds correct to me.  pwc_net_chairs_things.caffemodel has been trained using Chairs->Things and should have a smaller EPE on Chairs than pwc_net.caffemodel that has been trained through Chairs->Things->Sintel+HD1K+KITTI2015
3. The does not sound normal to me. What is the AEPE on Chairs/Sintel?

Best regards,
Deqing
